build/BUILD.clogger_embear

```diff
@@ -0,0 +1,7 @@
+licenses(["notice"])  # MIT
+
+cc_library(name="clogger_embear",
```

Comment:
let's call this `logger` so it can be used as `@com_github_embear_logger//:logger`